### ISSUE TITLE
[question] Namespace sandbox

### ISSUE THREAD

**Issue by anchev** [Eloston/ungoogled-chromium#519]
### Problem description

The README speaks about namespace sandbox. I have never heard of these. With a little searching I found [this info](https://stackoverflow.com/a/39722086) based on what it seems my system supports namespaces.

My questions are:

1. Considering the above - what should I actually do?
2. Why is all this namespace business important and why not simply run `chrome`? (please ELI5)

**Comment by csmk** [comment]
> 1. Considering the above - what should I actually do?

As your system supports namespaces, then there's nothing to do about it.

> 2. Why is all this namespace business important and why not simply run `chrome`? (please ELI5)

It's not an ELI5, but I'm sure you'll understand the summary: https://chromium.googlesource.com/chromium/src/+/lkgr/docs/linux_sandboxing.md

**Comment by Eloston** [comment]
@csmk Thanks.

@anchev Please let us know if this doesn't answer the question.

**Comment by csmk** [comment]
> Does that mean I should simply run `chrome` and not worry about any other executables?

Yes.

From the link above:

> _You can see which sandboxes are currently engaged by looking at `chrome://sandbox` (renderer processes) and `chrome://gpu` (gpu process)._

**Comment by csmk** [comment]
> `chrome://sandbox/` is telling me 'No' for SUID Sandbox and 'Yama LSM Enforcing'.

Those are just alternative modes that should be used when namespaces aren't available.

> `chrome://gpu/` shows Sandboxed: false and in its **Log Messages** I read:

I need to investigate this further. I'm also getting that first line in your log.

**Comment by anchev** [comment]
Thanks for explaining.

> I need to investigate this further. I'm also getting that first line in your log.

I hope you can share your findings.

**Comment by Eloston** [comment]
@csmk 

> I need to investigate this further. I'm also getting that first line in your log.

I've also been getting that error for many versions now (at least with the Debian builds). If you do find something, I'll be interested in hearing it :+1:

**Comment by csmk** [comment]
> I've also been getting that error for many versions now (at least with the Debian builds).

Yeah, I've seen that error way back, like ~62.x, but tbh, I never gave the proper attention to it.

Also, that error seems more common than I thought, it has been a bit hard get any useful information. There are many bug reports, but they always seems to be just co-related to it.

**Comment by Eloston** [comment]
@csmk I see, perhaps there's an open Chromium bug report somewhere if it's that prevalent. If you find a method to fix it (beyond just updating Chromium), let us know via a new issue. Thanks.
